### ISSUE TITLE
Quality of youtube videos

### ISSUE THREAD

**Issue by jakop345** [AndreMiras/tubedl#3]
It would be gret if we could select quality of youtube videos.

**Comment by AndreMiras** [comment]
Hi @jakop345,
At the moment we download the best quality available.
We try to stick to the Keep It Simple Stupid and Pareto principles and I don't see many use-cases where it would be interesting to download different quality. So we would probably not implement this feature in the short or medium term.
Thanks for your report.

**Comment by jakop345** [comment]
No quality is required because not every one has a great internet speed in their countries and in some fup limit is too small.
It might also reduce load on server because not every one want a 720p version

**Comment by AndreMiras** [comment]
Yes that's a valid use case, however I don't think 80% of users have bad internet connection.
Perhaps I'll change my mind and implement that feature one day. But at the moment I prefer to keep the code and the interface as simple as possible.
